js-miniapp-sdk/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CHANGELOG
 
+### In Progress (2020-XX-XX)
```

Comment:
Maybe you should also mention the update to `requestLocationPermission` since it will now request both the device and custom permission?

Reply:
Yeah, I have updated this part.